src/components/productView/productViewOpenShiftContainer.js

```diff
@@ -98,63 +103,70 @@ const ProductViewOpenShiftContainer = ({ t }) => {
 
     return (
       <React.Fragment key={`product_${productId}_${uomFilter}`}>
```

Comment:
optional: this works, but you could consolidate by moving the key prop to context provider